doc/package-notes.xml

```diff
@@ -233,18 +233,17 @@ $ nix-env -f '&lt;nixpkgs&gt;' -qaP -A eclipses --description
    <varname>jvmArgs</varname> is a list of arguments given to the JVM running
    the Eclipse. For example, say you wish to install the latest Eclipse
    Platform with the popular Eclipse Color Theme plugin and also allow Eclipse
-   to use more RAM. You could then add
+   to use more RAM. You could then add an overlay like
 <screen>
-packageOverrides = pkgs: {
-  myEclipse = with pkgs.eclipses; eclipseWithPlugins {
+self: super: {
+  myEclipse = with self.eclipses; eclipseWithPlugins {
```

Comment:
nit: `super.eclispeWithPlugins`.

Reply:
`eclipseWithPlugins` is in `eclipses`, not in nixpkgs top level. Should it be `super.eclipses.eclipseWithPlugins`, or left as is?

Reply:
Yes, it should come from `super`.